src/main/java/com/mixpanel/android/mpmetrics/MixpanelFCMMessagingService.java

```diff
@@ -424,6 +456,35 @@ protected static Notification makeNotificationSDK21OrHigher(Context context, Pen
             notificationTitle = "A message for you";
         }
 
+        if (null != buttonsJsonStr) {
+            try {
+                JSONArray buttonsArr = new JSONArray(buttonsJsonStr);
+                for (int i = 0; i < buttonsArr.length(); i++) {
+                    JSONObject buttonObj = buttonsArr.getJSONObject(i);
+
+                    // get button icon from name if one sent
+                    final String btnIconName = buttonObj.optString("icnm");
+                    int btnIcon = MixpanelFCMMessagingService.NotificationData.NOT_SET;
+                    if (null != btnIconName) {
```

Comment:
`btnIconName` will be empty if it doesn't exist

Reply:
ah optString returns `""` eh?